features/index_page.feature

```diff
@@ -0,0 +1,9 @@
+Feature: Webpage with name
+
+  Scenario: I visit the page
+    Given I visit the index page
+    Then I would like to see the text "Sophie is a student of Craft Academy"
+
+  Scenario: I visit Amber page
+    Given I visit the Amber page
```

Comment:
I would prefer if you could pass in a name in this step and store it as a variable.

Reply:
Ok